### PR TITLE
Feat/13 notifications

### PR DESCRIPTION
**Pull request by cy**
screen shot 2016-09-15 at 4 43 32 pm

### CODE REVIEW

**Comment by cy** [comment]
@shanewilson @alex-wilmer r?

File: theme/js/gdc-notifications.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      if (response.status >= 200 && response.status < 300) {
+        return response;
+      } else {
+        var error = new Error(response.statusText);
+        error.response = response;
+        throw error;
+      }
+    };
+
+    return new Promise(function(resolve, reject) {
```
Comment (alex-wilmer) [inline theme/js/gdc-notifications.js]: why Promise here? fetch already returns a promise

Comment (cy) [inline theme/js/gdc-notifications.js]: removed

File: theme/js/gdc-notifications.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        return response.json();
+      }).then(function(json) {
+        resolve(json.data);
+      });
+    });
+  };
+
+  var readDismissedCookie = function() {
+    var cookieValue = document.cookie.replace(/(?:(?:^|.*;\s*)gdc-dismissed-notifications\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+    var dismissedIds = JSON.parse(cookieValue || '[]');
```
Comment (cy) [inline theme/js/gdc-notifications.js]: but JSON parsing '[]' gives []! I think it looks fine but ok

File: theme/js/gdc-notifications.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      <span class="header-banner-dismiss"> \
+          Dismiss <span class="fa fa-close" aria-hidden="true"></span>\
+      </span>';
+
+    notifications.forEach(function(n) {
+      var banner = container.append('div')
+      .classed('header-banner', true)
+      .classed('enter', true)
+      .html(bannerHTML);
+      setTimeout(function() { banner.classed('enter', false), 1});
```
Comment (cy) [inline theme/js/gdc-notifications.js]: opps, thanks. I wanted it set to the minimal amount of time

**Pull request merged**